### ISSUE TITLE
Surface level review

### ISSUE THREAD

**Issue by johnurbanik** [arena-tools/covid-public#7]
Hey all,

Very cool work here - definitely useful to have a dashboard that compares multiple models to one another. The phase space model is a very interesting way of looking at things! I actually started to play a bit with visualization of this type the [other night](https://github.com/understand-covid/epimodelingtoolkit/tree/master/parameters/infection%20rate).

A quick breakdown of some concerns with the models.

### SIR: 

You're using a Kermack-McKendrick model, the simplest form of SIR model out there. It feels like more of a toy model to be illustrative than anything else, but I'll note at the least that:

1. Latency is a critical factor in modeling this disease, as it drastically changes how control measures work. I'd extend at the very least to an SEIR model, which also requires pushing the R0 up (as your serial interval increases). Current belief on R0 is closer to the 5.7 range (https://wwwnc.cdc.gov/eid/article/26/7/20-0282_article?deliveryName=USCDC_333-DM25287)
2.  Recovery time is different than infectious time in the current environment. People isolating when they have symptoms changes these dynamics.
3. Asymptomatic/presymptomatic/prodromal spread is a huge characteristic of COVID. Even if these cases don't contribute directly to deaths, they make control substantially harder - even in countries with robust contact tracing, long chains of asymptomatic cases could result in a new deadly outbreak weeks after the last confirmed case in a region.
4. Implementing at least one kind of intervention into the model seems like a bare minimum to get this model operating in a similar class to the others. Something that allows changing the contact rate and recovery time at specific dates would be great.

### Curve fitting  (both phase space and IHME derived)

Both of these models are predicated on the idea that deceleration continues; i.e. that the measures are effective in eventually bringing r0 below 1 and that those measures are not let up at any point in the future. Carl Bergstrom has several pretty good breakdowns of the weaknesses of these types of models:

https://twitter.com/CT_Bergstrom/status/1245618003711946753
https://twitter.com/CT_Bergstrom/status/1247645708741566465
https://twitter.com/CT_Bergstrom/status/1246957709682806785

*tldr;* while case counts / death counts may be decelerating, there is no reason to assume that that deceleration will continue (or acceleration will not start) unless the herd immunity is reached. 

In China we've already seen cases where relaxation of lockdown results in a new acceleration in cases - the danger in using these models as a 'peak' forecaster is that they assume that the mitigation measures completely eradicate the epidemic or that those mitigation measures are never relaxed.

While purely statistical models are useful (and should definitely be developed in combination with the mechanistic ones), we need to be careful that assumptions made in the statistical models don't violate the underlying mechanics or otherwise limit the possible outcomes in unrealistic ways. Imports (and travel policy) make a huge difference here. This can be seen directly with Connecticut and New Jersey, but a few new case injections into an area that has been social distancing mildly but not had any imports  recently (and thus looks like it is doing well at controlling growth) could have a drastically different trajectory after a few imports or a single super-spreader.
